src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -643,7 +641,6 @@ public function next_token() {
 		return true;
 	}
 
-
```

Comment:
I won't ask to remove this, but it was intentionally added to give a little bit more separation between the `next_` methods and the query/read methods

Reply:
Oh, the two empty lines seemed out of place. I'm surprised PHPCS didn't complain, in other places that's a lint error.

I can restore the line or we could add an organizational comment, thoughts?